core/domain/question_services_test.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from core.domain import question_domain
+from core.domain import question_services
+from core.domain import user_services
+from core.platform import models
+from core.tests import test_utils
+
+(question_models,) = models.Registry.import_models([models.NAMES.question])
+memcache_services = models.Registry.import_memcache_services()
+
+class QuestionServicesUnitTest(test_utils.GenericTestBase):
+    """Test the question services module."""
+
+    def setUp(self):
+        """Before each individual test, create dummy user."""
+        super(QuestionServicesUnitTest, self).setUp()
+
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+
```

Comment:
You don't need this blank line, nor the one that appears two below. The calls here are all related.

Reply:
Updated.